dashboard/app/controllers/project_versions_controller.rb

```diff
@@ -16,4 +16,22 @@ def get_token
     headers['csrf-token'] = form_authenticity_token
     return head :ok
   end
+
+  def project_commits
+    user_storage_id, project_id = storage_decrypt_channel_id(params[:channel_id])
+    project_owner = User.find_by(id: user_id_for_storage_id(user_storage_id))
+    return render :not_acceptable unless project_owner
+    return render :forbidden unless can?(:project_commits, ProjectVersion.new, project_owner, project_id)
+    commits = ProjectVersion.where(project_id: project_id).where.not(comment: '').order(created_at: :desc)
```

Comment:
We want it sorted by most recent last, so I think that's `:asc` right?

Reply:
Oh, sorry I thought it was most recent first. My bad -- I'll change!